### PR TITLE
Update zh trans kubeadm_upgrade_diff.md

### PR DESCRIPTION
**Pull request by jiazxjason**
Update zh trans kubeadm_upgrade_diff.md

### CODE REVIEW

**Comment by tengqm** [comment]
/hold
please base your translation on `https://raw.githubusercontent.com/kubernetes/website/master/content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_diff.md`

**Comment by jiazxjason** [comment]
> /hold
> please base your translation on `https://raw.githubusercontent.com/kubernetes/website/master/content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_diff.md`

My translation is based on https://github.com/kubernetes/website/blob/master/content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_diff.md, which is just the link you put

**Comment by tengqm** [comment]
@jiazxjason You are viewing the markdown rendered by the github markdown engine. That means you are not translating the original source where embedded HTML sources are used.

**Comment by jiazxjason** [comment]
@tengqm please review, thanks

**Comment by tengqm** [comment]
Thanks. Can you interleave the translated content with the original content?
By interleaving, we can easy track changes in the future and it simplifies review.

Instead of 
```
<!--
original content ... 10s of paragraphs
-->
translated content ... 10s of paragraphs
```
please do

```
original content
<!-- original content 1 to be translated -->
translated content 1
<!-- original content 2 to be translated -->
translated content 2
...
```

see [this file](https://raw.githubusercontent.com/kubernetes/website/master/content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md) for an example.

**Comment by jiazxjason** [comment]
@tengqm  Sure, please review

**Comment by tengqm** [comment]
The preview page doesn't look good. The second table is not shown.
https://deploy-preview-22271--kubernetes-io-master-staging.netlify.app/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_diff/

You need to tweak the HTML comment before the second table. Currently the `-->` sequence is incorrectly processed as HTML code, if you examine the HTML generated.

**Comment by jiazxjason** [comment]
@tengqm I don't know what happened on the preview page. It's ok on my local typora and I'm sure the second table is included and not surrounded by '<--'.

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_diff.md [hunk 1]
```diff
@@ -1,141 +1,184 @@
...
+<td colspan="2">--scheduler-manifest string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值："/etc/kubernetes/manifests/kube-scheduler.yaml"</td>
 </tr>
 <tr>
-  <td></td><td style="line-height: 130%; word-wrap: break-word;">调度程序清单的路径</td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">调度程序清单的路径</td>
 </tr>
 
+</tbody>
+</table>
 <!--
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_diff.md:144]: Please try insert an empty line before line 144.

**Comment by jiazxjason** [comment]
@tengqm wow! it works now

**Pull request merged**